src/pages/storage/StoragePoolOptionHeader.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react";
+const StoragePoolOptionHeader: FC = () => {
+  const columns = ["name", "driver", "usage"];
+
+  if (columns.length < 2) {
+    return null;
+  }
+
+  const headerSegments = columns.map((column) => {
+    return (
+      <span key={column} className="resource u-no-margin--bottom">
+        {column === "imageType" ? "image type" : column}
```

Comment:
What does the image type mean here, can it be removed?

Reply:
I see this is duplicating ResourceOptionHeader. I think a better solution is to generalize that component, so we can reuse it here and for permissions.